CHANGES.rst

```diff
@@ -43,6 +43,9 @@ astropy.io.votable
 
 astropy.modeling
 ^^^^^^^^^^^^^^^^
+- ``astropy.modeling.functional_models.Moffat1D.fwhm`` and
```

Comment:
no  need for the full name,  saying only ``Moffat1D.fwhm`` and ``Moffat2D.fwhm`` will do.